roles/ceph-rgw-loadbalancer/tasks/pre_requisite.yml

```diff
@@ -1,11 +1,59 @@
 ---
 - name: install haproxy and keepalived
   package:
-    name: ['haproxy', 'keepalived']
+    name: haproxy
     state: present
   register: result
   until: result is succeeded
 
+============ WIP
+
+- name: install keepalived
+  package:
+    name: keepalived
+    state: present
+  register: result
+  when: keepalived_package_source is undefined
+  until: result is succeeded
+
+- name: check if specified keepalived version already exists
+  shell: "ls /usr/sbin/ | grep keepalived-{{ keepalived_version }}"
```

Comment:
This could probably be simpler, grep isn't required. Something such as:

```
command: "ls /usr/sbin/keepalived-{{ keepalived_version }}"
```

Will require `failed_when: false` and `changed_when: false`.

Reply:
done